### DISCUSSION TITLE
Why not StateFlow instead of Value?

### DISCUSSION THREAD

**Discussion by YektaDev** [arkivanov/Decompose#69] [Q&A] [answered]
Hi.
I'm impressed by the idea behind this library and found it the best available approach to achieve multi-platform navigation, so thanks for the effort! I was trying to get started. While reading the documentation, I noticed the `Value` class:

> [Value](https://github.com/arkivanov/Decompose/blob/master/decompose/src/commonMain/kotlin/com/arkivanov/decompose/value/Value.kt) - is a multiplatform way to expose streams of states. It contains the value property, which always returns the current state. It also provides ability to observe state changes via subscribe/unsubscribe methods. There is [MutableValue](https://github.com/arkivanov/Decompose/blob/master/decompose/src/commonMain/kotlin/com/arkivanov/decompose/value/MutableValueBuilder.kt) which is a mutable variant of Value.

I'm wondering if there are some reasons for not using Kotlin's `StateFlow` in order to achieve this functionality?

**Comment by arkivanov** [comment, marked as answer]
Hello. This looks like a duplicate of https://github.com/arkivanov/Decompose/discussions/46.
